windows_containers/windows-containers-release-notes-2-x.adoc

```diff
@@ -17,6 +17,18 @@ The release notes for Red Hat OpenShift for Windows Containers tracks the develo
 
 You must have a subscription to receive support for the Red Hat WMCO. Deploying Windows container workloads in production clusters is not supported without a subscription. If you do not have a subscription, you can use the community WMCO, a distribution that lacks official support. Request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
 
+[id="wmco-2-0-2"]
+== Release notes for Red Hat Windows Machine Config Operator 2.0.2
+
+Issued:2021-07-XX
+
+The WMCO 2.0.2 is now available with bug fixes. The components of the WMCO were released in link:https://access.redhat.com/errata/RHSA-XXXX:YYYY[RHSA-XXXX:YYYY].
+
+[id="wmco-2-0-2-bug-fixes"]
+=== Bug fixes
+
+* Previously, when you created a Windows pod, the status would remain stuck in "ContainerCreating." This bug was due to the `BoundServiceAccountTokenVolume` option being enabled by default with {product-title} 4.8. This option caused projected volumes being attached to all of the pods. Finally, the `RunAsUser` option was added to the `SecurityContext` when executing the `os.Chown()` method, resulting in failure. Now, after upgrading to WMCO 2.0.2 prior to upgrading to {product-title} 4.8, he status updates to "Running" once the Windows pod is active. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1975553[**BZ#1975553**])
```

Comment:
Use backticks for `ContainerCreating` because it's a literal value that customers can view in the terminal.

For the second sentence--and this might be a dud idea, but consider--maybe "This bug was due to {product-title} 4.8 setting the `BoundServiceAccountTokenVolume` option by default."

Third sentence...maybe "This option caused the Operator to attach the projected volume to all the pods." (Not sure that's actually true--just my attempt at a more active voice.)

Lastly, the `Running` status should be in bacticks too instead of quotation marks--it's a literal value and all that.

Oh, change "once" to "after" or "when", from the ISG:
> Use only to mean "one time". Do not use as a conjunction to mean "after" or "when".